pkg/driver/deployment/dirigent.go

```diff
@@ -70,6 +70,11 @@ func deployDirigent(function *common.Function, controlPlaneAddress string) {
 		"requested_memory":    {strconv.Itoa(function.MemoryRequestsMiB)},
 	}
 
+	if busyLoopOnColdStart {
```

Comment:
For better readability one might want to add an else statement with set the value explicitly to 0. Not sure though.... You decide